spaces/novadefi/index.json

```
@@ -0,0 +1,15 @@
+{
+  "key": "novadefi",
+  "name": "novadefi.com",
+  "chainId": 1,
+  "decimals": 18,
+  "symbol": "NMT",
+  "defaultView": "all",
+  "address": "0xd9A6803f41A006CBf389f21e55D7A6079Dfe8DF3",
+  "token": "0xd9A6803f41A006CBf389f21e55D7A6079Dfe8DF3",
+  "core": [
+    "0x26efbfe4f747322AEb106a47Aa206e7c83EF10d6",

```

Please remove comma.
```suggestion
    "0x26efbfe4f747322AEb106a47Aa206e7c83EF10d6"

```